lib/awful/hotkeys_popup/keys/qutebrowser.lua

```diff
@@ -0,0 +1,187 @@
+---------------------------------------------------------------------------
+--- Qutebrowser hotkeys for awful.hotkeys_widget
+--
+-- @author Simon Désaulniers sim.desaulniers@gmail.com
+-- @copyright 2017 Simon Désaulniers
+-- @module awful.hotkeys_popup.keys.qutebrowser
+---------------------------------------------------------------------------
+
+local hotkeys_popup = require("awful.hotkeys_popup.widget")
+
+local qutebrowser_rule_any = {class={"qutebrowser", "qutebrowser"}}
```

Comment:
See https://awesomewm.org/doc/api/libraries/awful.rules.html#rules for the documentation.

I've meant to use:
```
["Qutebrowser: copying/yanking"]    = { rule = {instance = "qutebrowser"} },
```

or

```
local qutebrowser_rule = {instance={"qutebrowser"}}
…
["Qutebrowser: copying/yanking"]    = { rule = qutebrowser_rule},
```

(basically matching the WM "instance" instead of "class" (both are in WM_CLASS, and usually different, but for qutebrowser apparently the same))